testing/moosefs/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Michael Pirogov <vbnet.ru@gmail.com>
 # Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
 pkgname="moosefs"
-pkgver="3.0.105"
+pkgver="3.0.107"
 pkgrel=1
```

Comment:
pkgrel needs to be reset to 0 when pkgver is changed.

Reply:
thanks, done ;)